R/routes.R

```diff
@@ -80,7 +80,7 @@
 #' route_cyclestreet(c(-2, 52), c(-1, 53), "fastest")
 #' }
 route_cyclestreet <- function(from, to, plan = "fastest", silent = TRUE, pat = cyclestreet_pat(),
```

Comment:
@Robinlovelace i'd suggest not having `cyclestreet_pat()` as the default for the `pat` parameter. someone e.g., could pass in `NULL`, or the data.frame `mtcars`, etc. It's not likely, but could happen. Instead leave it as `NULL`  then use `cyclestreet_pat()` internally, before you do the HTTP request, so that you fail early with helpful message about the PAT if they don't supply it/etc. 

same for `route_graphhopper `